### PR TITLE
adding pages

### PR DESCRIPTION
**Pull request by njbp92**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Adoptable Dogs Page </title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+</head>
+<body>
+    <script src="script/script.js"> </script>
+  <header>
+    <div class="header">
+      <img class= "dogface" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/header-image.jpg" alt= "brown dog looking through cracked blinds with sunlight beaming through">
```
Comment (jimmynono) [inline blog.html]: These image links don't work. They will need to be relative links to your project. If you look on your gihub pages you will see all of your images are broken

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+           <img class="logo" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/adoptadog-logo.svg" alt= "adoptadog white logo">
+        <h2 class="logo" style="color:rgb(132,187,66);"> Seattle's premiere dog adoption agency </h2>
+    </div>
+    </header>
+
+    <div class="buttons">
```
Comment (jimmynono) [inline blog.html]: Watch your indenting. It's off all over the file

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    <title> Adoptable Dogs Page </title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+</head>
+<body>
+    <script src="script/script.js"> </script>
+  <header>
+    <div class="header">
+      <img class= "dogface" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/header-image.jpg" alt= "brown dog looking through cracked blinds with sunlight beaming through">
+           <img class="logo" src="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/adoptadog-logo.svg" alt= "adoptadog white logo">
+        <h2 class="logo" style="color:rgb(132,187,66);"> Seattle's premiere dog adoption agency </h2>
```
Comment (jimmynono) [inline blog.html]: No inline styles. Do it all from the CSS file

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+           </p>
+          <p> Duis ultricies lacus sed turpis tincidunt id. Ipsum a arcu cursus vitae congue. Ipsum consequat nisl vel pretium lectus. A diam maecenas sed enim ut sem viverra. Nulla facilisi nullam vehicula ipsum a arcu cursus vitae. Dictum non consectetur a erat nam at lectus urna. Et molestie ac feugiat sed lectus vestibulum mattis. Scelerisque fermentum dui faucibus in. At imperdiet dui accumsan sit amet nulla facilisi morbi. Dictumst quisque sagittis purus sit amet volutpat consequat. Sagittis orci a scelerisque purus semper. Scelerisque varius morbi enim nunc faucibus a. Commodo elit at imperdiet dui accumsan sit amet nulla. Morbi quis commodo odio aenean sed adipiscing diam.
+         </p>
+          </section>
+          <section class= "Walk-Multiple">
```
Comment (jimmynono) [inline blog.html]: It's generally good practice to use all lowercase letters for CSS classes with a - seperating the words. Also, watch for extra spaces. You don't need  a space after the = sign.

```
<section class="walk-multiple">
```

File: dogs.html [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+      </div>
+
+   	       <section class= "adopt">
+           <h2 class="adopt" style="color:rgb(9,60,96);">Here are all of our dogs!</h2>
+
+              <br>
+              <div class="box">
+                    <img class="dog" src ="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/murphy-card.jpg" alt= "older brown ridgeback">
+                    <h2  style="color:rgb(9,60,9);"> Murphy </h2>
+                    <strong> Cost to Adopt:</strong> $123.45 <br>
```
Comment (jimmynono) [inline dogs.html]: Love that you're using strong!

File: dogs.html [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+                  <br>
+                 <input type="submit" value="Adopt">
+              </div>
+                  <div class="box">
+                    <img class="dog" src ="C:/Users/JoJo/Desktop/html200/adopt-a-dog-spring2019/images/jack-card.jpg" alt= "white and brown american foxhound">
+                    <h2 style="color:rgb(9,60,96);"> Jack </h2>
+                      <strong> Cost to Adopt:</strong> $123.45 <br>
+                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Rutrum tellus pellentesque eu tincidunt tortor aliquam nulla. Et malesuada fames ac turpis. Sit amet massa vitae tortor. Sed lectus vestibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt.</p>
+                        <br>
+                    <input type="submit" value="Adopt">
```
Comment (jimmynono) [inline dogs.html]: A button might be a better choice for these

File: styles/stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  width: auto;
+}
+.logo {
+  position: relative;
+  margin: 5px;
+  width: 33%;
+}
+.buttons {
+  font-family: inherit;
+  font-size: 100%;
```
Comment (jimmynono) [inline styles/stylesheet.css]: I'm not a huge fan of using percents for font-size. Embrace rems or ems (tip: rems are easier to use)

File: styles/stylesheet.css [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+  text-align: left;
+  padding-bottom: 150px;
+}
+.blogpic {
+  width: 20%;
+  float: left;
+  position: relative;
+  justify-content: space-around;
+}
+.Travel {
```
Comment (jimmynono) [inline styles/stylesheet.css]: All of these blog posts can be combined to one css class and use that same css class for each of the blogs

File: blog.html (hunk 1 above)
Comment (njbp92) [reply blog.html]: This part confused me. I noticed that after I pushed everything. How do I fix the links?

Comment (jimmynono) [reply blog.html]: You should use relative paths in all of your work, it should work on your computer, as well as on a server (like github pages). Hopefully your editor didn't autofill the c://.... part.

```
      <img class= "dogface" src="images/header-image.jpg" alt= "brown dog looking through cracked blinds with sunlight beaming through">
```

If you update your links to this, it should work. Nice descriptive alt tag!

File: styles/stylesheet.css [hunk 9]
```diff
@@ -0,0 +1,144 @@
+body {
+  background-color: rgb(236,236,236);
+  align-items: center;
+  margin: 1rem;
+}
+
+.dogface {
```
Comment (jimmynono) [inline styles/stylesheet.css]: This isn't quite the right way to do this. We'll cover the proper use of background images in a later module.

File: styles/stylesheet.css [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+  display: flex;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: rgb(236, 236,236);
+  width: 100%;
+  position: absolute;
+  margin-top: 45px;
+  }
+
+.red-shirt {
```
Comment (jimmynono) [inline styles/stylesheet.css]: not a fan of floats for this use. Maybe use grid or flexbox on the wrapping element?

**Pull request closed without merging**